staging/vhost-device-gpu/src/virt_gpu.rs

```diff
@@ -55,6 +55,121 @@ fn sglist_to_rutabaga_iovecs(
     Ok(rutabaga_iovecs)
 }
 
+pub trait VirtioGpuBackend
```

Comment:
I would suggest renaming this trait to `VirtioGpu` and naming the actual implementation `RutabagaVirtioGpu`. What do you think?

Reply:
I think it make sense. I can change it then.